src/modules/pulseaudio.cpp

```diff
@@ -18,6 +18,12 @@ namespace modules {
     // Load configuration values
     m_interval = m_conf.get(name(), "interval", m_interval);
 
+    auto click_right = m_conf.get(name(), "click-right", ""s);
+
+    if (!click_right.empty()) {
+      m_builder->cmd(mousebtn::RIGHT, click_right);
```

Comment:
This won't work. You need to call `m_builder->cmd` in the `get_output` method. The text module has a good example for that.